### PR TITLE
Fix warnings introduced by .NET 6.0 SDK and use latest AppVeyor images

### PR DESCRIPTION
**Pull request by thomaslevesque**

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <LangVersion>latest</LangVersion>
     <Nullable>enable</Nullable>
     <AnalysisMode>AllEnabledByDefault</AnalysisMode>
+    <CheckEolTargetFramework>false</CheckEolTargetFramework>
```
Comment (blairconrad) [inline Directory.Build.props:11]: Had to image to learn this wasn't all about "end-of-line".

**Review by blairconrad — APPROVED** [review]
Me, I approve. Appveyor less so…

**Comment by thomaslevesque** [comment]
> Me, I approve. Appveyor less so…

Yes, I saw... It's clearly related to what I was telling you about, but on my machine it worked fine on the command line, so I'm not sure what's going on here

File: Directory.Build.props (hunk 1 above)
Comment (thomaslevesque) [reply Directory.Build.props:11]: end of line? I thought it was end of life

**Comment by blairconrad** [comment]
Thanks, @thomaslevesque! I

**Pull request merged**